sirepo/raydata/databroker.py

```diff
@@ -6,6 +6,8 @@
 from pykern.pkdebug import pkdp
 import databroker
 
+_CACHED_CATALOGS = {}
```

Comment:
Any reason not to use a PKDict?

Reply:
no reason, other than it wasn't already imported to the module. I've updated it now